src/Sulu/Bundle/HttpCacheBundle/CacheLifetime/CacheLifetimeRequestEnhancer.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\HttpCacheBundle\CacheLifetime;
+
+use Symfony\Component\HttpFoundation\RequestStack;
+
+class CacheLifetimeRequestEnhancer
+{
+    const ATTRIBUTE_KEY = '_cacheLifetime';
+
+    /**
+     * @var RequestStack
+     */
+    private $requestStack;
+
+    public function __construct(RequestStack $requestStack)
+    {
+        $this->requestStack = $requestStack;
+    }
+
+    public function setCacheLifetime(int $cacheLifetime)
+    {
+        $request = $this->requestStack->getCurrentRequest();
+
+        if (!$request) {
+            return;
```

Comment:
should we store it in a private variable when no current request is set? Not sure if the cache information could be interested in some kind of a CLI script 🤔 

/cc @wachterjohannes

Reply:
Since the rest of the caching information is probably also not available on the CI I guess that is not a big deal. I guess if we want to make that available in the CLI, then we should make the final cache lifetime available, not this intermediate one.